src/Server/Provision/ServerProvisioner.php

```diff
@@ -412,14 +432,12 @@ private function getRaidLevel()
     }
 
     /**
-     * @param array $choices
+     * @param array|null $addons
      *
      * @return array
      */
-    private function addons(array $choices)
+    private function addons(array $addons = null)
```

Comment:
if this were null, array_filter would fail. This should always be an array

Reply:
I can check again if you're sure but pretty certain that works perfectly, when I left the values blank it would 500 on me (which is extra confusing cause it just loads forever doing nothing, have to check console to see 500), but when I added this it started working perfectly.

Reply:
Ya I just tested it again, 500 and loads forever when I remove it, works perfectly when I add it back. If you think its jank and want me to change it so it never even hits that function if its null, or early returns on null I can do that.